portfolio/src/main/webapp/style.css

```diff
@@ -1,3 +1,93 @@
+/* Styling for comment delete button */
+.delete {
+    color: white;
+    background-color: #d9534f;
+    border-color: #d43f3a;
+}
+
+/* Break between commenter name and the content */
+.comment_break {
+    margin: 10px;
+}
+
+/* Styling for button in comment */
+.comment_button { 
+    display: inline-block;
+    padding: 1px 4.2px;
+    font-weight: normal;
+    text-align: center;
+    vertical-align: middle;
+    touch-action: manipulation;
+    border: 1px solid transparent;
+    white-space: nowrap;
+    font-size: 10px;
+    line-height: 1.42857143;
```

Comment:
…oddly specific…  the Internet tells me this is what Bootstrap uses.  Any reason you're adopting that particular value for this?

Reply:
hahahah I saw some formatting for an icon button I liked and didn't want to import bootstrap so I may have copied the bootstrap.... I am definitely gonna update it